sdk/identity/azure-identity/src/main/java/com/azure/identity/implementation/util/PowerShellUtil.java

```diff
@@ -0,0 +1,47 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.identity.implementation.util;
+
+/**
+ * Utility class for powershell auth related ops .
+ */
+public class PowerShellUtil {
+    public static String getPwshCommand(String tenantId, String scope, String sep) {
```

Comment:
Would we be better off having this as a proper PowerShell script that we ship as part of resources in the project?

Would simplify this by making this a call like

Get-AccessToken -TenantId tenantId -Scope scope

Reply:
This would also allow this logic to be more portable.